app/models/dhs1171_pdf.rb

```diff
@@ -29,4 +53,10 @@ def client_data
       signature_date: snap_application.signed_at,
     }
   end
+
+  def add_cover_sheet_to_completed_form
+    system(
+      "pdftk #{COVERSHEET_PDF} #{completed_filename} cat output #{output_filename}",
```

Comment:
ugh I am kind of regretting my comments from before because it feels like it is not necessarily clear out of `completed` and `output` which is the more final version. Can always be renamed another time, though...